src/components/ui/Toast/toast.scss

```diff
@@ -0,0 +1,121 @@
+@import "src/styles/scaffold";
+
+[data-fs-toast] {
+  // --------------------------------------------------------
+  // Design Tokens for Toast
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-toast-width                        : calc(100% - (2 * var(--fs-spacing-3)));
+
+  --fs-toast-min-height                   : var(--fs-spacing-10);
+  --fs-toast-padding                      : var(--fs-spacing-1);
```

Comment:
The padding seems off, the proposal uses a .5rem of spacing. Try to use the `--fs-spacing-1 for it` :)
|Current|Proposed|
|-|-|
|<img width="445" alt="image" src="https://user-images.githubusercontent.com/11613011/168899835-9a6179dc-98af-4675-afd4-b26cd6e0ca68.png">|<img width="441" alt="image" src="https://user-images.githubusercontent.com/11613011/168899918-23882f9f-4923-4947-ab67-e339aa156643.png">|